frontend/src/components/EntertainmentButton/EntertainmentButton.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { ENTERTAINMENT } from "./constants";
+
+import "./styles.css";
+import { MenuButton } from "../MenuButton/MenuButton";
```

Comment:
Let's configure project to follow for imports like `src/components/` instead of `../../components/`